sql/triggers.sql

```diff
@@ -45,6 +45,7 @@ CREATE TRIGGER char_delete
 	BEFORE DELETE ON chars
 	FOR EACH ROW
 BEGIN
+    DELETE FROM `char_blacklist` WHERE `charid_owner` = OLD.charid;
```

Comment:
`charid_owner`?

And just for consistency, either use tabs or change the whole file to spaces.

Reply:
Ah yeah my edit does tabs to spaces. And yes in the blacklist table the charid is charid_owner

Reply:
Right, forgot it was named something different in blacklist.

cOnSIStEncY

Reply:
If you want we can do a overhaul on it and change it all to charid?

Reply:
It's fine, just do tabs to spaces and we're golden.